app/frontend/state.ts

```diff
@@ -196,7 +197,7 @@ const initialState: State = {
   sendTransactionSummary: {
     // this should be called only transactionSummary
     type: TxType.SEND_ADA,
-    address: null,
+    address: null as any,
```

Comment:
Is this also related to the `// TODO avoid casting to any` comment?

Reply:
yes, as mentioned in the comment above, I removed that